openstack/keystone/Chart.lock

```diff
@@ -1,10 +1,10 @@
 dependencies:
 - name: mariadb
   repository: https://charts.eu-de-2.cloud.sap
-  version: 0.7.11
+  version: 0.8.0
```

Comment:
Will this upgrade mariadb? I am concerned, because we never figured out why mariadb failed last time, and decided to keep it at the old version until we manage to investigate.

Reply:
disregard this

Reply:
I have used the value from the master branch. Why is it shown as a diff?
- https://github.com/sapcc/helm-charts/blob/master/openstack/keystone/Chart.yaml#L17
- https://github.com/sapcc/helm-charts/blob/master/openstack/keystone/Chart.lock#L4